File: Frontend/Logic/OstiumLogic.Tests/GamesManagement/Context/Fixtures.cs
using B20.Architecture.Contexts.Api;
using GamesManagement.Api;
using Ostium.Logic.Tests.GamesManagement.Fixtures;

namespace Ostium.Logic.Tests.GamesManagement.Context
{
    public class GameManagementMocks: ContextModule
    {
        public void Apply(ContextBuilder builder)
        {
            builder.SetImpl<GamesManagementApi, GamesManagementApiMock>();
        }
    }
}

File: Backend/lib/src/testFixtures/kotlin/com/github/bratek20/ostium/carddrawer/context/Fixtures.kt
package com.github.bratek20.ostium.carddrawer.context

import com.github.bratek20.architecture.context.api.ContextBuilder
import com.github.bratek20.architecture.context.api.ContextModule
import com.github.bratek20.ostium.carddrawer.api.CardDrawerApi
import com.github.bratek20.ostium.carddrawer.fixtures.CardDrawerApiMock

class CardDrawerMocks: ContextModule {
    override fun apply(builder: ContextBuilder) {
        builder.setImpl(CardDrawerApi::class.java, CardDrawerApiMock::class.java)
    }
}